libbeat/processors/condition.go

```diff
@@ -42,6 +42,10 @@ type WhenProcessor struct {
 	p         Processor
 }
 
+type ValuesMap interface {
```

Comment:
Note to myself: Figure out why this change is needed and if it could happen in a separate PR.